### DISCUSSION TITLE
how to emulate Apple Silicon on Intel based mac?

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#5907] [macOS]
I would like to emulate a Mac running on Apple Silicon chip on my Mac with an Intel chip.
is it possible?

**Comment by ghost** [comment]
I'm guessing it's not, as I'm pretty sure its only Intel on Apple Silicon but not the other way round. Maybe? I'm not sure

**Comment by tifasoftware** [comment]
Give it 10-15 years and someone will be bound to find a way to emulate it

**Comment by bonna79** [comment]
> I would like to emulate a Mac running on Apple Silicon chip on my Mac with an Intel chip.
> is it possible?

In questi giorni mi sono imbattuto nella tua stessa richiesta e per ora Windows 7 x86 e Windows xp , vanno senza problemi , anche audio e scheda video , ovviamente UEFI deve essere disabilitato. Per Macosx ho scoperto che bisogna farlo partire con UEFI attivo e impostare una sorta di file UEFI , altrimenti non parte . L'unica cosa che non sono riuscito a far funzionare è proprio un OS X , perché dovrei far partire software x86 a 32 bit , cosa che oramai con gli ultimi OS X tutto gira solo a 64bit . 
Ho trovato questo link che spiega il discorso emulazione OS X dentro OS X silicon , ma non ho capito come e dove impostare i file : https://developer.apple.com/documentation/virtualization/installing_macos_on_a_virtual_machine

L'unica cosa che devo provare è usare altre macchine virtuali per emulare OS X x86 , ma sono mv non open e preferirei capire se posso farlo anche con software libero . Io credo che si possa fare . Teniamoci aggiornati .
Grazie e buone feste 

These days I have come across the same request as you and for now Windows 7 x86 and Windows xp , go without problems , even audio and video card , obviously UEFI must be disabled. For Macosx I found that you have to start it with UEFI active and set up some sort of UEFI file, otherwise it won't start. The only thing I couldn't get to work is just an OS X, because I should start 32-bit x86 software, which now with the latest OS X everything only runs at 64bit.

I found this link explaining the OS X emulation speech inside OS X silicon , but I didn't understand how and where to set up the files : https://developer.apple.com/documentation/virtualization/installing_macos_on_a_virtual_machine

The only thing I have to try is to use other virtual machines to emulate OS X x86 , but they are mv not open and I would prefer to understand if I can do it even with free software . I believe it can be done. Let's keep up to date.

Thank you and happy holidays
